src/CSS/footer.css

```diff
@@ -0,0 +1,26 @@
+footer {
+  margin-top: 3rem;
+  width: 100%;
+  height: 100px;
+  background-color: purple;
+  color: white;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 20px;
+  font-weight: bolder;
+  position: relative;
+}
+
+/* footer */
+
+footer > * {
+  margin: 0;
+  padding: 0;
```

Comment:
Reservation design
- [ ] Kindly make sure the positions of the elements are similar. :smiley: 
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197341008-c2a24fad-759f-4412-9d33-95baba9f98ff.png)

Template design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197340994-cae1635f-69c7-4903-af9b-025a92092e9b.png)